### DISCUSSION TITLE
Russian translation

### DISCUSSION THREAD

**Discussion by Blueberryy** [LorisYounger/VPet#393] [Translate]
I would like to help with that, what files do i need to translate?

**Comment by LorisYounger** [comment]
Thank you for your support. I am sorry that the game itself will not add languages other than English, Simplified Chinese, and Traditional Chinese.
Because I cannot confirm the text content and there are currently many versions of translation documents in the workshop, I cannot choose to make one of them an official translator.
You can upload to steam workshop and add [Human Translation] in title that let everyone know the advantages of your translation.
You can also post it in the discussion. Let everyone know that there is a Russian translation available
I will also release a special announcement in the future to inform everyone of the existing community translation content and location
In the future, if I understand and start to use the translation platform, I will notify you separately.
Thank you for support of VPET project.

**Comment by Blueberryy** [comment]
> Thank you for your support. I am sorry that the game itself will not add languages other than English, Simplified Chinese, and Traditional Chinese.
> Because I cannot confirm the text content and there are currently many versions of translation documents in the workshop, I cannot choose to make one of them an official translator.
> You can upload to steam workshop and add [Human Translation] in title that let everyone know the advantages of your translation.
> You can also post it in the discussion. Let everyone know that there is a Russian translation available
> I will also release a special announcement in the future to inform everyone of the existing community translation content and location
> In the future, if I understand and start to use the translation platform, I will notify you separately.
> Thank you for support of VPET project.

Understadable. Can i translate Store Page and Workshop main page & tags on Steam then?

**Comment by LorisYounger** [comment]
Of course, thank you for your translation

This is workshop tag of russian and englisn

[workshop_tags_1920960_russian.json](https://github.com/LorisYounger/VPet/files/12909654/workshop_tags_1920960_russian.json)

[workshop_tags_1920960_english.json](https://github.com/LorisYounger/VPet/files/12909656/workshop_tags_1920960_english.json)

**Comment by Blueberryy** [comment]
> Of course, thank you for your translation
> 
> This is workshop tag of russian and englisn
> 
> [workshop_tags_1920960_russian.json](https://github.com/LorisYounger/VPet/files/12909654/workshop_tags_1920960_russian.json)
> 
> [workshop_tags_1920960_english.json](https://github.com/LorisYounger/VPet/files/12909656/workshop_tags_1920960_english.json)

What do you think of localizing this?
TF2 has this custom workshop item translated for their workshop.
![2023_10_15_0vl_Kleki](https://github.com/LorisYounger/VPet/assets/36592509/28ee7b94-f51b-4da2-936d-58647f3443c0)

**Comment by LorisYounger** [comment]
> > Of course, thank you for your translation
> > This is workshop tag of russian and englisn
> > [workshop_tags_1920960_russian.json](https://github.com/LorisYounger/VPet/files/12909654/workshop_tags_1920960_russian.json)
> > [workshop_tags_1920960_english.json](https://github.com/LorisYounger/VPet/files/12909656/workshop_tags_1920960_english.json)
> 
> What do you think of localizing this? TF2 has this custom workshop item translated for their workshop. ![2023_10_15_0vl_Kleki](https://user-images.githubusercontent.com/36592509/275330480-28ee7b94-f51b-4da2-936d-58647f3443c0.png)

![image](https://github.com/LorisYounger/VPet/assets/34806646/51ff3e8e-2190-4d94-820c-7e04e308a64e)
![image](https://github.com/LorisYounger/VPet/assets/34806646/925f2544-006d-46d7-ad4e-7dea6cf0d0d8)

It seems that Steam did not prepare a translation for the module name

**Comment by Blueberryy** [comment]
can you give me some more info about "lowtext", "clicktext" and "selecttext"? what do they stands for?

**Comment by LorisYounger** [comment]
clicktext#Click text:|
lowtext#Low status text:|
selecttext#Options Chat Text:|

**Comment by Blueberryy** [comment]
Can you describe Low status text? Like it's a feature that shows text when you minimize VPet?

**Comment by LorisYounger** [comment]
Low status text is words spoken when feeling thirsty or hungry

**Comment by LorisYounger** [comment]
Some example https://github.com/LorisYounger/VPet/blob/main/VPet-Simulator.Windows/mod/0000_core/lang/en/Text2307.lps

**Comment by Blueberryy** [comment]
alright, here's russian tags
[workshop_tags_1920960_russian.json](https://github.com/LorisYounger/VPet/files/13030779/workshop_tags_1920960_russian.json)
I hope you will send main page text too and store page text.

**Comment by LorisYounger** [comment]
main page text is same as https://github.com/LorisYounger/VPet/blob/main/README_en.md#introduction

**Comment by Blueberryy** [comment]
Okay but can you export this text to translation file?
![image](https://github.com/LorisYounger/VPet/assets/36592509/0edbf002-92e9-4514-84f9-09a9e5768ec6)

**Comment by Blueberryy** [comment]
> main page text is same as https://github.com/LorisYounger/VPet/blob/main/README_en.md#introduction

It has no short description of store page that contains ``Feeling like the computer desktop is too dull? Need something cute to heal oneself? Come and try this new completely free and open source V-Pet simulator. A cute and adorable pet that supports various feeding interactions to accompany you in game learning is definitely your best choice!``

**Comment by Blueberryy** [comment]
and most of file is not as steam store page description... it would be awesome if you just export localization files of your pages on github

**Comment by LorisYounger** [comment]
I took a look and it seems like I updated the description of Steam later on.
This is what Steam originally described
```
VPet is a desktop pet software(game) that supports various interactive. it is open source, free and support Steam workshops

This game is independent from the built-in desktop pet (tutorial) program of VUP Simulator, if you like it, you can also add VUP Simulator to your wishlist
[url]https://store.steampowered.com/app/1352140/_/[/url]

[h2]Lots of interactions and animations[/h2]

More than 200 Type of Animation, 

[b]Some Animation Examples[/b]:
[b]touch the head & Body[/b]
[img]{STEAM_APP_IMAGE}/extras/ss2.gif[/img]
[img]{STEAM_APP_IMAGE}/extras/ss1.gif[/img]
[b]bring up[/b]
[img]{STEAM_APP_IMAGE}/extras/ss8.gif[/img]
[b]dance[/b]
[img]{STEAM_APP_IMAGE}/extras/ss19.gif[/img]
[b]climb the wall[/b]
[img]{STEAM_APP_IMAGE}/extras/ss7.gif[/img]

See more image resources details in open source docs

[h2]Free[/h2]

The game is completely free!
The main purpose of the game is to promote the VUP-Simulator, and the character in the game is the protagonist of the virtual anchor simulator.

[h2]Open Source[/h2]

The game is open source on github, welcome to put forward your own ideas, ideas or participate in the development!
You can also modify the code to make your own table pet! (Although most of the content supports the workshop, is no need to modify the code)
project address: [url=https://github.com/LorisYounger/VPet]github-LorisYounger/VPet[/url]

[h2]Support Steam Workshop[/h2]

The game supports the Steam Workshop, you can make animations or interactions with other characters and table pets, and upload them to the creative workshop to share with more people.

The creative workshop supports adding/modifying the following content
The creative workshop supports adding/modifying the following content
[list] [*] Table pet animation
[*] Items/food/beverages, etc
[*] Table pet work
[*] Speech text
[*] Theme
[*] Code plugin - Add content to desktop pets by writing code
[list] [*] Add new animation logic/display scheme (eg: l2d/spin, etc.)
[*] Add new features (alarm clock/dashboard, etc.)
[*] Almost omnipotent [/list]
[/list]

[h2]Feedback & Suggestions & Contact Us[/h2]

If you have any suggestions or comments, you can contact me in Steam store Comments/Community, Github Issue or email me
```

**Comment by LorisYounger** [comment]
> Okay but can you export this text to translation file? ![image](https://user-images.githubusercontent.com/36592509/278802245-0edbf002-92e9-4514-84f9-09a9e5768ec6.png)

This is 
```
VPET Simulator WorkShop
Go and try to Find More Pet!
```

**Comment by Blueberryy** [comment]
> I took a look and it seems like I updated the description of Steam later on. This is what Steam originally described
> 
> ```
> VPet is a desktop pet software(game) that supports various interactive. it is open source, free and support Steam workshops
> 
> This game is independent from the built-in desktop pet (tutorial) program of VUP Simulator, if you like it, you can also add VUP Simulator to your wishlist
> [url]https://store.steampowered.com/app/1352140/_/[/url]
> 
> [h2]Lots of interactions and animations[/h2]
> 
> More than 200 Type of Animation, 
> 
> [b]Some Animation Examples[/b]:
> [b]touch the head & Body[/b]
> [img]{STEAM_APP_IMAGE}/extras/ss2.gif[/img]
> [img]{STEAM_APP_IMAGE}/extras/ss1.gif[/img]
> [b]bring up[/b]
> [img]{STEAM_APP_IMAGE}/extras/ss8.gif[/img]
> [b]dance[/b]
> [img]{STEAM_APP_IMAGE}/extras/ss19.gif[/img]
> [b]climb the wall[/b]
> [img]{STEAM_APP_IMAGE}/extras/ss7.gif[/img]
> 
> See more image resources details in open source docs
> 
> [h2]Free[/h2]
> 
> The game is completely free!
> The main purpose of the game is to promote the VUP-Simulator, and the character in the game is the protagonist of the virtual anchor simulator.
> 
> [h2]Open Source[/h2]
> 
> The game is open source on github, welcome to put forward your own ideas, ideas or participate in the development!
> You can also modify the code to make your own table pet! (Although most of the content supports the workshop, is no need to modify the code)
> project address: [url=https://github.com/LorisYounger/VPet]github-LorisYounger/VPet[/url]
> 
> [h2]Support Steam Workshop[/h2]
> 
> The game supports the Steam Workshop, you can make animations or interactions with other characters and table pets, and upload them to the creative workshop to share with more people.
> 
> The creative workshop supports adding/modifying the following content
> The creative workshop supports adding/modifying the following content
> [list] [*] Table pet animation
> [*] Items/food/beverages, etc
> [*] Table pet work
> [*] Speech text
> [*] Theme
> [*] Code plugin - Add content to desktop pets by writing code
> [list] [*] Add new animation logic/display scheme (eg: l2d/spin, etc.)
> [*] Add new features (alarm clock/dashboard, etc.)
> [*] Almost omnipotent [/list]
> [/list]
> 
> [h2]Feedback & Suggestions & Contact Us[/h2]
> 
> If you have any suggestions or comments, you can contact me in Steam store Comments/Community, Github Issue or email me
> ```

Here's translation of it:

```
VPet - это настольная программа (игра) с питомцами, поддерживающая различные виды интерактивного общения. Она имеет открытый исходный код, бесплатна и поддерживает мастерскую Steam

Эта игра не зависит от встроенной программы настольного питомца (программы обучения) VUP Simulator, если она вам понравилась, Вы также можете добавить VUP Simulator в свой список желаемого
[url]https://store.steampowered.com/app/1352140/_/[/url]

[h2]Множество взаимодействий и анимаций[/h2]

Более 200 типов анимации, 

[b]Примеры анимации[/b]:
[b]прикосновение к голове и телу[/b]
[img]{STEAM_APP_IMAGE}/extras/ss2.gif[/img]
[img]{STEAM_APP_IMAGE}/extras/ss1.gif[/img]
[b]поднимание/b]
[img]{STEAM_APP_IMAGE}/extras/ss8.gif[/img]
[b]танец[/b]
[img]{STEAM_APP_IMAGE}/extras/ss19.gif[/img]
[b]карабканье по стене[/b]
[img]{STEAM_APP_IMAGE}/extras/ss7.gif[/img]

Более подробную информацию о ресурсах изображений можно найти в документации по открытым источникам

[h2]Бесплатно[/h2]

Игра абсолютно бесплатна!
Основная цель игры - продвижение VUP-Simulator, а персонаж игры - главная героиня из неё.

[h2]Открытый исходный код[/h2]

Игра имеет открытый исходный код на Github, приглашаем выдвигать свои идеи, задумки или участвовать в разработке!
Вы также можете модифицировать код, чтобы сделать своего собственного настольного питомца! (Хотя большая часть содержимого поддерживается в мастерской, модифицировать код не требуется)
адрес проекта: [url=https://github.com/LorisYounger/VPet]github-LorisYounger/VPet[/url]

[h2]Поддержка мастерской Steam[/h2]

Игра поддерживает мастерскую Steam, вы можете создавать анимации или взаимодействия с другими персонажами и настольными питомцами и загружать их в творческую мастерскую, чтобы поделиться ими с большим количеством людей.

Творческая мастерская поддерживает добавление/изменение следующего контента
[list] [*] Анимация настольных питомцев
[*] Предметы/пища/напитки и т.д.
[*] Работа с питомцев за столом
[*] Текст речи
[*] Тема
[*] Кодовый плагин - добавление содержимого на настольных питомцев с помощью написания кода
[list] [*] Добавление новых логик анимаций/схем отображения (например, l2d/spin и т.д.)
[*] Добавление новых функций (будильник/приборная панель и т.д.)
[*] Практически без ограничений [/list]
[/list]

[h2]Отзывы, предложения и связь с нами[/h2]

Если у вас есть какие-либо предложения или замечания, вы можете связаться со мной через обсуждения сообщества Steam, создав Issue на Github или написать мне по электронной почте
```



> 
> This is
> 
> ```
> VPET Simulator WorkShop
> Go and try to Find More Pet!
> ```

Here's translation of it:

```
Мастерская симулятора VPET
Опробуйте больше питомцев!
```

**Comment by Blueberryy** [comment]
also there's duplicated  text at the end of steam store page description

**Comment by Blueberryy** [comment]
and send here short description of store page please

**Comment by LorisYounger** [comment]
> and send here short description of store page please

```
Feeling like the computer desktop is too dull? Need something cute to heal oneself? Come and try this new completely free and open source V-Pet simulator. A cute and adorable pet that supports various feeding interactions to accompany you in game learning is definitely your best choice!
```

**Comment by Blueberryy** [comment]
> > and send here short description of store page please
> 
> ```
> Feeling like the computer desktop is too dull? Need something cute to heal oneself? Come and try this new completely free and open source V-Pet simulator. A cute and adorable pet that supports various feeding interactions to accompany you in game learning is definitely your best choice!
> ```

i had to simplify and remove some text to fit 300 characters limit,
but here's translation to it:
```
Рабочий стол слишком скучен? Или же вам просто одиноко? Опробуйте этот новый совершенно бесплатный симулятор V-Pet с открытым исходным кодом. Милый и очаровательный питомец с различными взаимодействиями при кормлении, который будет постоянно вместе с вами, определённо изменит вашу жизнь!
```

**Comment by Blueberryy** [comment]
ah and i did typo in text formatting, please copy and paste fixed text:

```
[b]Примеры анимации[/b]:
[b]прикосновение к голове и телу[/b]
[img]{STEAM_APP_IMAGE}/extras/ss2.gif[/img]
[img]{STEAM_APP_IMAGE}/extras/ss1.gif[/img]
[b]поднимание[/b]
[img]{STEAM_APP_IMAGE}/extras/ss8.gif[/img]
[b]танец[/b]
[img]{STEAM_APP_IMAGE}/extras/ss19.gif[/img]
[b]карабканье по стене[/b]
[img]{STEAM_APP_IMAGE}/extras/ss7.gif[/img]
```
